sources/packages/web/src/views/institution/InstitutionProfile.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-  <header-navigator title="Manage institutions" subTitle="Manage Profile" />
-  <Message severity="info">
-    Please notice that the read-only information below is retrieved from your
-    BCeID account and it is not possible to change it here. If any read-only
-    information needs to be changed please visit
-    <a href="https://www.bceid.ca/" target="_blank" rel="noopener">bceid.ca</a>.
-  </Message>
-  <full-page-container>
-    <institution-profile-form
-      :profileData="institutionProfileModel"
-      @submitInstitutionProfile="updateInstitution"
-    ></institution-profile-form>
-  </full-page-container>
+  <v-container>
+    <header-navigator title="Manage institutions" subTitle="Manage Profile" />
+    <Message severity="info">
```

Comment:
This message is not part of the form.io definition?

Reply:
did u mean banners? as per the discussion with @abschwenker banner is not part of this story as there is an adjustments needed to be done in the design